frontend/src/components/Game.js

```diff
@@ -88,6 +93,8 @@ export default function Game({gameId, clientUsername, initialPlayer, playerNames
                                   diceResult={diceResult} selectedWord={selectedWord}
                                   currentPlayer={currentPlayer} clientUsername={clientUsername}
                                   sendWebSocketMessage={sendMessage}/>
+    }else if(gamePhase == GamePhase.GAME_OVER){
+        return <EndGamePage currentPlayer={currentPlayer}/>
```

Comment:
Using currentPlayer to store the winning player at the end of the game is fine I think 👍 But the prop of EndGamePage should be named correctly (i.e. winningPlayer, not currentPlayer).